hw/system/occamy/src/occamy_soc_ctrl/occamy_soc_reg.hjson.tpl

```diff
@@ -101,6 +96,23 @@
         }
       ]
     }
+    { name: "NUM_QUADRANTS"
+      desc: "Number of quadrants per chip."
+      swaccess: "ro"
+      hwaccess: "none"
+      hwqe:     "true",
+      hwext:    "true",
+      fields: [
+        {
+          bits: "31:0"
+          resval: ${nr_s1_clusters}
```

Comment:
Based on the name and description I would this expect to be `nr_s1_quadrant` instead of  `nr_s1_clusters`?

Reply:
Good point. This is a typo :| I presume during my test I made some superfluous accesses that went nowhere. I will fix any issues in bulk later.